engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```diff
@@ -421,7 +421,11 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
         permissionsInfo = new JSONObject(IOUtils.toString(is));
       } catch (IOException ex) {
         // ignore if file is not found
-        LOG.info("Exception in accessing permissions.json", ex);
+        if (LOG.isDebugEnabled()) {
```

Comment:
Yes, it all applies if/when "the feature becomes widely used". For now, my preferred way is to explicitly check whether the file exists or not and silently (debug level message is OK) disable the feature if it does not. If file exists, can we log info level message that the feature will be enabled?